Source/carthage/Bootstrap.swift

```diff
@@ -26,12 +26,35 @@ public struct BootstrapCommand: CommandType {
 					carthage.println(formatting.bullets + "No Cartfile.resolved found, updating dependencies")
 					return project.updateDependencies(shouldCheckout: options.checkoutAfterUpdate)
 				}
-
+				
+				let checkDependencies: SignalProducer<(), CarthageError>
+				if let depsToUpdate = options.dependenciesToUpdate {
+					checkDependencies =
+						project
+						.loadResolvedCartfile()
+						.flatMap(.Concat) { resolvedCartfile -> SignalProducer<(), CarthageError> in
+							let resolvedDependencyNames = resolvedCartfile.dependencies.map { $0.project.name }
+							let unresolvedDependencyNames = Set(depsToUpdate).subtract(resolvedDependencyNames)
```

Comment:
One thing we forgot to account for: case differences. Dependency names should be case insensitive.